Mortgage-loan-calculator/src/app/calculator-form/calculator-form.component.html

```diff
@@ -1,329 +0,0 @@
-                      [intrestPaid]="calculateResultsDto.totalInterestPaid"
-                      [agreementFee]="calculateResultsDto.agreementFee"
-
-                    ></app-pie-chart
-                    ><mat-divider [vertical]="true"></mat-divider>
-
-                  </div>
-
-                  <div class="form2">
-                    <div class="grid-container">
-                      <div class="grid-item-2">
-                        <div class="amountDisplay">
-                          <div class="align-image">
-                            <mat-icon class="brightness-1-icon"
-                              >brightness_1</mat-icon
-                            >
-
-
-                            <span matPrefix>Loan amount</span>
-                          </div>
-                          <div class="amount-positioning">
-                            {{ calculateResultsDto.maxLoan }}
-                          </div>
-                        </div>
-
-
-                      </div>
-                      <div class="grid-item-2">
-                        <div class="amountDisplay">
-                          <div class="align-image">
-                            <mat-icon class="brightness-2-icon"
-                              >brightness_1</mat-icon
-                            >
-
-
-                            <span matPrefix>Total interest paid</span>
-                          </div>
-                          <div class="amount-positioning">
-                            {{ calculateResultsDto.totalInterestPaid }}
-                          </div>
-                        </div>
-
-
-                      </div>
-
-                      <div class="grid-item-2">
-                        <div class="amountDisplay">
-                          <div class="align-image">
-                            <mat-icon class="brightness-3-icon"
-                              >brightness_1</mat-icon
-                            >
-
-
-                            <span matPrefix>Agreement fee</span>
-                          </div>
-                          <div class="amount-positioning">
-                            {{ calculateResultsDto.agreementFee }}
-                          </div>
-                        </div>
-                      </div>
-                      <div class="grid-item-2">
-                        <div class="amountDisplay">
-                          <span matPrefix>Total payment sum</span>
-
-                          <strong>
-                            {{ calculateResultsDto.totalPaymentSum }}</strong
-                          >
-                        </div>
-
-
-                      </div>
-                    </div>
-                    <div class="button-container">
-                      <button
-                        mat-button
-                        type="submit"
-                        class="my-button submit-button"
-                        (click)="onSubmit()"
```

Comment:
Make one method to handle click event. Also it probably should be form submission handler, not button click handling.